lib/ansible/modules/clustering/consul_acl.py

```diff
@@ -245,16 +245,20 @@ def yml_to_rules(module, yml_rules):
                 rules.add_rule('event', Rule(rule['event'], rule['policy']))
             elif ('query' in rule and 'policy' in rule):
                 rules.add_rule('query', Rule(rule['query'], rule['policy']))
+            elif ('agent' in rule and 'policy' in rule):
+                rules.add_rule('agent', Rule(rule['agent'], rule['policy']))
+            elif ('node' in rule and 'policy' in rule):
+                rules.add_rule('node', Rule(rule['node'], rule['policy']))
             else:
-                module.fail_json(msg="a rule requires a key/service/event or query and a policy.")
+                module.fail_json(msg="a rule requires a key/service/event/query/agent or node and a policy.")
```

Comment:
the phrasing of the message makes it seem like `node` is fundamentally different from the rest of the rule types, but it isn't. also, can we reference the full set of RULE_TYPES rather than hardcoding them here? I'd suggest something like:
`"a rule requires one of %s and a policy." % ('/'.join(RULE_TYPES))`